app/models/validators/aaib_report_validator.rb

```diff
@@ -9,5 +9,5 @@ class AaibReportValidator < SimpleDelegator
   validates :summary, presence: true
   validates :body, presence: true, safe_html: true
 
-  validates :date_of_occurrence, presence: true, date: true
+  validates :date_of_occurrence, date: true, allow_blank: true
```

Comment:
When I sketched this out, and played with it in dev, I wondered if the validation message should be tweaked as well so that it says "date of occurrence is required unless it is a Safety study".